src/main/java/org/sonarlint/intellij/ui/resolve/MarkAsResolvedOptionPanel.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2023 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.ui.resolve
+
+import com.intellij.ui.JBColor
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.components.JBPanel
+import com.intellij.ui.components.JBRadioButton
+import com.intellij.util.ui.JBUI
+import org.sonarsource.sonarlint.core.clientapi.backend.issue.IssueStatus
+import java.awt.Font
+import java.awt.GridBagConstraints
+import java.awt.GridBagLayout
+import javax.swing.BorderFactory
+class MarkAsResolvedOptionPanel (
+    status: IssueStatus,
+) : JBPanel<MarkAsResolvedOptionPanel>(GridBagLayout()) {
+
+    val statusRadioButton = JBRadioButton()
+
+    init {
+        statusRadioButton.actionCommand = status.name
```

Comment:
I just discovered this "action command" concept, I am not sure it's the most appropriate way to achieve that